frontend/src/lib/i18n/locales/en.json

```diff
@@ -1,4 +1,7 @@
 {
+  "about": {
+		"title": "About"
+	},
```

Comment:
These 2 lines seem to be the only ones that use tabs instead of spaces.
We should stick to spaces.

Do you maybe have this option disabled?
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/74595a2a-9901-4603-b88c-8464eec5158c)

It looks like we sort of have tabs turned on for these files 🤔, but my tabs turn into spaces anyway, so I'm guessing this isn't really doing anything 🤷.
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/59909a3c-4636-4e45-901c-e9121ae9cd4a)

Reply:
I just fixed it. Thanks for pointing that out!
https://github.com/sillsdev/languageforge-lexbox/pull/859/commits/c27e01e17c9115c3d47998a0c08010dfaa2511d1

Reply:
@psh0078 I'd look at setting up https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig as that should follow all the rules automatically.

Reply:
@hahn-kev I set it up and it works. Thanks!